code/modules/cmc/crew.dm

```diff
@@ -400,8 +410,9 @@ Crew Monitor by Paul, based on the holomaps by Deity
 
 		for(var/entry in entries[holomap_z[uid]])
 			//can only be our z, so i'm not checking that, only if we have a pos
-			if(entry[1]) addCrewMarker(user, entry[1], entry[2], entry[3], entry[4], entry[5], entry[6], entry[7], entry[8], entry[10])
-
+			if(entry[1])
```

Comment:
Is this about weather or not the list is here at all? 
Also make it `entry[ENTRY_SEE_X]`

Reply:
its supposed to check if we got a pos so we should make a crewmaker. altough that is also checked in addcrewmarker